src/containers/Dashboard.vue

```diff
@@ -18,12 +18,13 @@
           @openPlan="openPlan"
           @openTools="openTools"
           @editProfile="editProfile"
```

Comment:
And lastly this event reciever that calls `editProfile` can be deleted. Don't remove the `editProfile` method itself though, as your `openProfile` method in this file calls it!